components/EntityAttributeValue.js

```diff
@@ -32,5 +32,14 @@ const formatImage = (key) => {
       case 'float':
         //TODO: Find a more accurate representation of float
         return Number(item.value_double);
+      case 'custom':
+        // Some custom values may be a stringified array/object
+        // Not sure if using a try/catch block is good practice
+        // but this is the only way I can pull this off :(
+        try {
+            return JSON.parse(item.value_long_string);
```

Comment:
I suggest we let the plugin handle how it will be rendered in the API. Probably 90% of the time it would be a simple JSON.parse, but we let the programmer have the freedom to override it.

```suggestion
      const Plugin = plugin(customName); //retrieve custonName somehow
      const plugin = new Plugin();
      plugin.toJSON(item.value_long_string); //handles the rendering of the API;
```